pkg/operators/ebpf/ebpf.go

```diff
@@ -191,6 +193,23 @@ type ebpfInstance struct {
 	wg sync.WaitGroup
 }
 
+type Map struct {
+	Name string
+	Type string
+}
+
+type Program struct {
+	Section string
+	Source  string
+}
+
+type VariableSpec struct {
+	Name   string
+	Offset uint64
+	Size   uint64
+	Map    string
+}
```

Comment:
Those 3 types don't need to be exported, they could be named `extraInfoMap`, `extraInfoProgram` and `extraInfoVariable`.